packages/xrpl/test/integration/transactions/depositPreauth.test.ts

```diff
@@ -155,77 +155,4 @@ describe('DepositPreauth', function () {
     },
     TIMEOUT,
   )
-
-  it(
-    'Invalid multiple of auth and unauth provided',
-    async () => {
-      const subjectWallet = await generateFundedWallet(testContext.client)
-
-      const credentialCreateTx: CredentialCreate = {
-        TransactionType: 'CredentialCreate',
-        Account: testContext.wallet.classicAddress,
-        Subject: subjectWallet.classicAddress,
-        CredentialType: stringToHex('Test Credential Type'),
-      }
-
-      await testTransaction(
-        testContext.client,
-        credentialCreateTx,
-        testContext.wallet,
-      )
-
-      const credentialAcceptTx: CredentialAccept = {
-        TransactionType: 'CredentialAccept',
-        Account: subjectWallet.classicAddress,
-        Issuer: testContext.wallet.classicAddress,
-        CredentialType: stringToHex('Test Credential Type'),
-      }
-
-      await testTransaction(
-        testContext.client,
-        credentialAcceptTx,
-        subjectWallet,
-      )
-
-      const authorizeCredentialObj: AuthorizeCredential = {
-        Credential: {
-          Issuer: testContext.wallet.classicAddress,
-          CredentialType: stringToHex('Test Credential Type'),
-        },
-      }
-
-      const wallet2 = Wallet.generate()
-      await fundAccount(testContext.client, wallet2)
-      const tx: DepositPreauth = {
-        TransactionType: 'DepositPreauth',
-        Account: testContext.wallet.classicAddress,
-        AuthorizeCredentials: [authorizeCredentialObj],
-        UnauthorizeCredentials: [authorizeCredentialObj],
-      }
-      await testTransaction(testContext.client, tx, testContext.wallet)
```

Comment:
This will not succeed because of the presence of both `AuthorizeCredentials` and `UnauthorizeCredentials`. However, if you look inside testTransaction function, it only expects a `tesSUCCESS` return code. This might have been the cause of the error in the CI/CD.

I don't expect any changes to the code from this comment, I'm just brain-storming here

Reply:
Yup, would have needed additional logic beyond the scope of this PR to make negative cases work. Am leaving this out for now until we all decide it would be good to have later on